Gemfile

```diff
@@ -16,9 +14,16 @@ gem 'bootstrap', '~> 4.1.1'
 gem 'jquery-rails'
 gem 'devise'
 gem 'pundit'
-gem 'grape'
 gem 'rubocop'
 
+# grape
+gem "grape"
```

Comment:
Consider using `grape-swagger` gem. It should be much simpler for testing, you will also have autogenerated documentation for your API 🙂